Library/Homebrew/formula.rb

```diff
@@ -1063,6 +1063,18 @@ def std_cmake_args
     ]
   end
 
+  # Returns the path to a make 3.81 or newer binary.
+  # Returns the system make on Leopard and newer, and the
+  # path to the actually-installed make on Tiger or older.
+  def make_path
```

Comment:
I'm happy to move this when we get it working, but in the meantime I'd like to keep it here so it works like how it does in Tigerbrew - or move it into the FileUtils extensions.